src/components/Filter/Filter.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Status } from '../../types/Status';
+
+interface Props {
+  currentStatus: Status;
+  handleFilterAll: () => void;
+  handleFilterActive: () => void;
+  handleFilterCompleted: () => void;
+}
+
+export const Filter: React.FC<Props> = React.memo(({
+  currentStatus,
+  handleFilterAll,
+  handleFilterActive,
+  handleFilterCompleted,
+}) => {
+  return (
```

Comment:
when you don't have any logic inside component and just return jsx then you can delete the word `return`

and just return jsx in round brackets
`}) => (`